doc/Circles-and-Privacy.md

```diff
@@ -60,19 +60,19 @@ This is a trust issue you need to be aware of.
 No software in the world can prevent your friends from leaking your confidential and trusted communications.
 Only a wise choice of friends.
 
-But it isn't as clear cut when dealing with GNU Social and other network providers.
+But it isn't as clear-cut when dealing with GNU Social and other network providers.
 If you look at the Contact Edit page for any person, we will tell you whether or not they are members of an insecure network where you should exercise caution.
 
 Once you have created a post, you can not change the permissions assigned.
 Within seconds it has been delivered to lots of people - and perhaps everybody it was addressed to.
-If you mistakenly created a message and wish you could take it back, the best you can do is to delete it.
-We will send out a delete notification to everybody who received the message - and this should wipe out the message with the same speed it was initially propagated.
-In most cases it will be completely wiped from the Internet - in under a minute.
+If you mistakenly created a message and wish to take it back, the best you can do is delete it.
+We will send out a delete notification to everybody who received the message - and this should wipe out the message with the same speed as it was initially propagated.
+In most cases, it will be completely wiped from the Internet - in under a minute.
 Again, this applies to Friendica networks.
-Once a message spreads to other networks, it may not be removed quickly and in some cases it may not be removed at all.
+Once a message spreads to other networks, it may not be removed quickly, and in some cases, it may not be removed at all.
 
-In case you haven't yet figured this out, we are encouraging you to encourage your friends to use Friendica - because all these privacy features work much better within a privacy-aware network.
-Many of the other social networks Friendica can connect to have no privacy controls.
+In case you still need to figure this out, we are encouraging you to encourage your friends to use Friendica - because all these privacy features work much better within a privacy-aware network.
+Many other social networks Friendica can connect to have no privacy controls.
```

Comment:
This is mostly a jab at GNU Social which lost relevance a while ago. I don't think we should keep it here, especially with the initial condescending tone.
```suggestion
```